public-docker-images/Dockerfile-debian-sid

```diff
@@ -0,0 +1,56 @@
+FROM golang:1.9.1 AS base
+MAINTAINER The Stripe Observability Team <support@stripe.com>
+
+
+FROM base AS deps
+RUN mkdir -p /build
+ENV GOPATH=/go
+RUN apt-get update
+RUN apt-get install -y zip
+RUN go get -u -v github.com/ChimeraCoder/gojson/gojson
+RUN go get -u -v github.com/golang/protobuf/protoc-gen-go
+RUN go get -d -v github.com/gogo/protobuf/protoc-gen-gofast
+WORKDIR /go/src/github.com/gogo/protobuf
+RUN git fetch
+RUN git checkout v0.5
+RUN go install github.com/gogo/protobuf/protoc-gen-gofast
+WORKDIR /go
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN go get -u -v golang.org/x/tools/cmd/stringer
+RUN wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip
+RUN unzip protoc-3.1.0-linux-x86_64.zip
+RUN cp bin/protoc /usr/bin/protoc
+RUN chmod 777 /usr/bin/protoc
+
+
+FROM deps AS src
+ARG BUILD_REF=master
+RUN mkdir -p /go/src/github.com/stripe
+WORKDIR /go/src/github.com/stripe
+RUN git clone --depth 1 https://github.com/stripe/veneur
+WORKDIR /go/src/github.com/stripe/veneur
+RUN git fetch --depth 1 origin ${BUILD_REF}:docker/${BUILD_REF}
+RUN git checkout docker/${BUILD_REF}
+
+
+FROM src AS test
+RUN go test -race -v -timeout 60s -ldflags "-X github.com/stripe/veneur.VERSION=$(git rev-parse HEAD) -X github.com/stripe/veneur.BUILD_DATE=$(date +%s)" ./...
+
+
+FROM test AS build
+RUN go build -a -v -ldflags "-X github.com/stripe/veneur.VERSION=$(git rev-parse HEAD)" -o /build/veneur ./cmd/veneur
+RUN go build -a -v -ldflags "-X github.com/stripe/veneur.VERSION=$(git rev-parse HEAD)" -o /build/veneur-emit ./cmd/veneur-emit
+
+
+FROM debian:sid AS release-prep
+RUN apt-get update
+RUN apt-get -y install ca-certificates
+
+
+FROM release-prep AS release
+WORKDIR /veneur/
+EXPOSE 8126/UDP 8126/TCP 8127/TCP 8128/UDP
+COPY --from=release-prep /etc/ca-certificates.conf /etc/ca-certificates.conf
```

Comment:
Why do we need to copy these certificates over?

Reply:
(Actually, I had done this just to test and it seemed to work, but the package contains more files than just that -- so I'd prefer to do the apt hack instead in this case, or just let the cache files exist -- will check it out)

Reply:
Turns out that Debian bundles a handy script in the docker image that cleans up after installs anyway!
`docker run --rm -it veneur:debian cat /etc/apt/apt.conf.d/docker-clean`:

```
DPkg::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
APT::Update::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };

Dir::Cache::pkgcache "";
Dir::Cache::srcpkgcache "";
```